uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/manage-leave-requests.html

```diff
@@ -93,7 +93,7 @@
         </div>
         <div class="checkbox">
           <label>
-            <input type="checkbox" ng-change = "vm.refresh()"
+            <input type="checkbox" ng-change="vm.refresh(null, true)"
```

Comment:
if you know that it will have to go back to page number 1, the pass "1" explicitly